src/plugins/intel_cpu/src/transformations/snippets/tpp/op/factory.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "factory.hpp"
+#include "eltwise.hpp"
+#include "reduce.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "ov_ops/type_relaxed.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace tpp {
+namespace op {
+namespace {
+struct CustomPowerStaticBuilder : public TPPNodeFactory::TPPCustomBuilder {
+    CustomPowerStaticBuilder() : TPPNodeFactory::TPPCustomBuilder() {
+        matcher = [](const std::shared_ptr<ov::Node>& n) {
+            std::set<float> supported_power{-1, 2, 0.5};
+            const auto& power_static = ov::as_type_ptr<const snippets::op::PowerStatic>(n);
+            return power_static && supported_power.count(power_static->get_power());
+        };
+        builder = [](const std::shared_ptr<ov::Node>& n) {
+            const auto& power_static = ov::as_type_ptr<snippets::op::PowerStatic>(n);
+            OPENVINO_ASSERT(power_static, "Attempt to create TPP node from unsupported input in power_static_builder");
+            const auto power = power_static->get_power();
+            const auto& input = n->get_input_source_output(0);
```

Comment:
Really minor :)
We can use `input_value` to get a parent output
```suggestion
            const auto& input = n->input_value(0);
```